contrastscan-remote/src/test/java/com/contrastsecurity/scan/GitHubCodeScanningAnalysisTests.kt

```diff
@@ -0,0 +1,14 @@
+package com.contrastsecurity.scan
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class GitHubCodeScanningAnalysisTests : BehaviorSpec({
+    given("an incomplete environment") {
+        `when`("i call it") {
+            then("it should do this") {
+                1 + 2 shouldBe 3
+            }
+        }
```

Comment:
I see you like spec-style unit tests :-)

Reply:
I wanted to try to write kotlin idiomatic tests and I was guessing the just using the JUnit4/5 libs and interfaces is not always the idiomatic way.   So I went looking for the standard kotlin test framework.  I came across the kotest framework which had some of these Spec-style testing language.  I've always liked thinking of tests in terms of expected "behavior" so I figured I'd give it a try.